python/xorbits/_mars/dataframe/sort/tests/test_sort_execution.py

```diff
@@ -375,6 +375,54 @@ def test_sort_index_execution(setup):
     expected = raw_gb_rolling_mean.sort_index()
     pd.testing.assert_frame_equal(result, expected)
 
+    # test multi-level DataFrame with multi-chunk
+    raw_index = pd.MultiIndex.from_arrays(np.random.randint(10, size=[4, 100]))
+    raw = pd.DataFrame(
+        {"foo": np.random.randint(100, size=100), "bar": range(100)}, index=raw_index
+    )
+    mdf = DataFrame(raw, chunk_size=20)
+
+    for level_ in [
```

Comment:
Better use `@pytest.mark.parametrize` to avoid multi for loops: xorbits._mars.dataframe.window.rolling.tests.test_rolling_execution.test_range_indexed_df_rolling_corr_execution

Reply:
Also you may want to test where `level` is a list of level names.

Reply:
done, thx